laser/laser_class.py

```diff
@@ -0,0 +1,14 @@
+import RPi.GPIO as GPIO
+
+class Laser:
```

Comment:
Document this using the [numpy documentation style](https://numpydoc.readthedocs.io/en/latest/format.html).

Reply:
Also, you may put the constructor documentation under the class declaration.